### ISSUE TITLE
WCAG 2.1 compliance using the Accessibility Insights extension errors

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#10666]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=11&t=31734&p=161661#p161661)

- Using https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni extension to generate the accessibility test
- Go to https://bryntum.com/products/grid/examples/basic/ and run the "FastPass" test for accessibility
- Receive a few improvements that needs to be made

[FastPass_20250124_BryntumGridBasicdemo.html.zip](https://github.com/user-attachments/files/18536369/FastPass_20250124_BryntumGridBasicdemo.html.zip)

"We are testing our application for WCAG 2.1 compliance using the Accessibility Insights extension (FastPass). Below is a list of identified accessibility issues related to Bryntum components, along with their impact and suggestions for resolution.

Since the library dynamically generates these elements, we are unable to handle these issues on our end within the project. We request your team to investigate and provide fixes or guidance.

Issues:

#b-textfield-1-input
Impact  Element is not focusable. Element has global ARIA attributes but is marked as presentational.
Suggestion by FastPass Elements marked as presentational should not have global ARIA attributes or tabindex.

#b-grid-1-normalSubgrid
Impact Element should have focusable content. Element itself should be focusable.
Suggestion by FastPass Ensure elements with scrollable content are accessible via keyboard.

#b-checkbox-1-input
Impact: ARIA role presentation is not allowed for the given element.
Suggestion by FastPass The role attribute must have a valid value for the element type.

#b-combo-1-input
Impact Invalid ARIA attribute value: aria-active descendant has an invalid value.
Suggestion by FastPass Ensure all ARIA attributes have valid values.

We’d appreciate it if they could be prioritized.
Thank you for your assistance!
![bryntum fast pass.PNG](https://forum.bryntum.com/download/file.php?id=32873)"

**Comment by ExtAnimal** [comment]
6.2 will address much of this.
